packages/hix/lib/Hix/New.hs

```diff
@@ -191,12 +192,13 @@ initProject conf = traverse_ createFile =<< newProjectFiles conf
 
 newProject :: NewProjectConfig -> M ()
 newProject conf = do
+  directory <- resolvePathUserDir conf.directory
   let name =
         fromMaybe
-          (ProjectName . Text.dropWhileEnd (== '/') . Text.pack . fromRelDir . dirname $ conf.directory)
+          (ProjectName . Text.dropWhileEnd (== '/') . Text.pack . fromRelDir . dirname $ directory)
           conf.name
-  when conf.printDirectory $ Console.out (pathText conf.directory)
-  local (\res -> res { cwd = conf.directory }) $
+  when conf.printDirectory $ Console.out (pathText directory)
```

Comment:
Forgot to say it last time – this should be printed after `initProject` was successful